lms/product/plugin/__init__.py

```diff
@@ -1,3 +1,4 @@
+# type: ignore
```

Comment:
Product/plugin code might need more time consuming refactor to typecheck. 

Just ignoring it for now to be able to enable check_untyped_defs